x-pack/plugins/endpoint/public/embeddables/resolver/models/process_event.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ProcessEvent } from '../types';
+
+export function isGraphableProcess(event: ProcessEvent) {
+  return eventType(event) === 'processCreated' || eventType(event) === 'processRan';
+}
+
+export function eventType(event: ProcessEvent) {
+  const {
+    data_buffer: { event_type_full: type, event_subtype_full: subType },
+  } = event;
+
+  if (type === 'process_event') {
```

Comment:
it would be nice to have this stuff scoped to particular transforms for endgame v0 events, mainly because none of this stuff is ECS compatible as it stands at the moment

Reply:
This will change once we use real, ECS compatible data, as opposed to sample data. We will hopefully do this soon.

Reply:
@andrewstucki I've never been able to get the streaming feature to produce a set of events that could be used for resolver. in all of my attempts, enough events are missing that I can't render a resolver graph from it. If you could provide some more realistic sample data we'll use it. Otherwise we can wait for @jonathan-buttner to finish a Resolver data generator. @jonathan-buttner is that ready?

Reply:
So, we should sit down together and see what you actually need, because the events shipped have literally everything in them that are part of our raw events--so if there's not enough data, we have a problem, because it means that the endpoint itself isn't shipping data compatible with what you're trying to visualize.

Reply:
sorry for the confusing explanation. the events have enough data, I just can't get the streaming feature to send all of the events.

Reply:
@oatkiller the data generator is close but still working out some issues. We haven't started on a backend implementation yet but that's next on the list.